.env.template

```diff
@@ -1,6 +1,12 @@
 #
 # Users that do not exist in the respective Docker image must be specified by their user ID
 #
+
+# System user & group
+FAF_FS_PUID=1003
+FAF_FS_PGID=1000
```

Comment:
These are UIDs on the host system, faf-stack should not make any assumptions.
Keep them `root`